cmake/Modules/FindAPL.cmake

```diff
@@ -71,4 +71,4 @@ int main() {
   exit((float)r != (float).1234);
 }" BLAS_USE_CBLAS_DOT )
   MESSAGE(STATUS "BLAS_USE_CBLAS_DOT: ${BLAS_USE_CBLAS_DOT}")
-ENDIF (APL_FOUND)
```

Comment:
are we removing newline at the end of the file?
can we revert this line change.

Reply:
Reverted.

Although I do prefer for files to have a new line at the end :smile: